Structural changes and adding information. (#25)
xournalpp/xournalpp.github.io: docs/guide/config.md

# Configuration

# Preferences

> Xournal ++ has multiple options that can be configured and customized. Most of these features can be found in the path **Edit> Preferences **.

Reviewer: ```suggestion
Xournal ++ has multiple options that can be configured and customized. Most of these features can be found under **Edit > Preferences**.
```

## Load / Save

### Autosaving 

Enable or disable automatic saving and set the time interval in which it will be done. If the document was previously saved in any folder, the autosave document will be in the same location as a hidden file. Otherwise the file will be saved in the path **~/.xournalpp/autosave**.

### Default Save Name

Name that will be proposed by default when using **Save As** option. Allows the use of placeholders.

### Autoloading Journals

TODO

## Input System

TODO

## Mouse

In this section you can set the behavior of the **middle and right mouse buttons** to associate tools with it. Each tool allows sub settings for thickness, color, stroke type, etc. if they are available.

## Stylus

> ### Preassure Sensitivity

Reviewer: ```suggestion
### Pressure Sensitivity
```
